csunplugged/tests/topics/views/test_curriculum_integration_view.py

```diff
@@ -164,11 +162,12 @@ def test_curriculum_integration_view_prerequisite_lessons_context_order(self):
             len(response.context["prerequisite_lessons"]),
             3
         )
+        print(response.context["prerequisite_lessons"])
         self.assertQuerysetEqual(
             response.context["prerequisite_lessons"],
             [
+                "<Lesson: Lesson 1 (8 to 10)>",
```

Comment:
not sure what you're asking, ``assertQuerysetEqual`` has ``ordered=True`` by default...so it is already ordered